analytic_amount_security/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2025 Moduon Team S.L.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl-3.0)
+{
+    "name": "Analytic amount security",
+    "summary": "Add a group to constrain which users can see what info on the analytic lines",
+    "version": "16.0.1.0.0",
+    "development_status": "Alpha",
+    "category": "Technical",
+    "website": "https://github.com/OCA/account-analytic",
+    "author": "Moduon, Odoo Community Association (OCA)",
+    "maintainers": ["chienandalu"],
```

Comment:
issue: add rafaelbn as explained in our [internal contribution guidelines](https://doc.moduon.team/doc/estandares-de-contribucion-gxxTTBnNnb#h-mantenedores).

```suggestion
    "maintainers": ["chienandalu", "rafaelbn"],
```